testdata/p4_16_errors_outputs/issue1541.p4-stderr

```diff
@@ -1,9 +0,0 @@
-issue1541.p4(25): [--Wwarn=shadow] warning: 'hash' shadows 'hash'
```

Comment:
why isn't this a warning? Isn't this declaration shadowed?

Reply:
It still is but I've moved the file from errors to samples - it doesn't output the `Recursive action call` error it did previously. I don't think this was an error because the inner `hash` is a method included in the header, whereas `action hash() { ... }` simply encapsulates it (semantically there should be no recursion here).

Reply:
I don't see a -stderr file for issue 1541 in samples_outputs.